test/dummies/v1beta2/vars.go

```diff
@@ -423,7 +423,7 @@ func SetDummyCAPIMachineVars() {
 		},
 		Spec: clusterv1.MachineSpec{
 			ClusterName:   ClusterName,
-			FailureDomain: pointer.String(Zone1.ID)},
+			FailureDomain: pointer.String("fd1")},
```

Comment:
do we not have a constant for this?

Reply:
We only have couple of places uses this `fd1`. And I changed this zone.id to 'fd1' since we have this multiple-end points feature added.